openwisp_monitoring/monitoring/base/models.py

```diff
@@ -73,7 +81,7 @@ class Meta:
             'field_name',
             'content_type',
             'object_id',
-            'extra_tags',
+            'main_tags',
```

Comment:
won't this cause issues if there are multiple metrics with different extra_tags but same main_tags?